docs/source/technical_tutorials/access/direct_access.ipynb

```diff
@@ -0,0 +1,1564 @@
+{
```

Comment:
<p>This won't work outside of MAAP ADE at all (well it might work on VEDA but that would have a different assumeRole)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/344/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>